makefile

```diff
@@ -0,0 +1,19 @@
+run:
+	python src/main.py bin/chromedriver ${EMAIL} ${PASSWORD}
```

Comment:
Is this what we wan't?
Should the chromedriver binary be an argument? is it going to change frequently?
Maybe its better to make it configurable.

Reply:
make will run only in linux envirment so the chrome driver will be fixed